01-intro/00-basics.md

```diff
@@ -0,0 +1,70 @@
+# Introduction to Event Driven Microservice Architectures
+
+## Introduction
+
+Imagine your CI/CD pipeline as a bustling kitchen, each task like chopping
+carrots or preheating the oven is an action. Now, think of Event Driven
+Microservice Architecture as the central communication hub in this kitchen. It's
+like a magical bulletin board capturing every action as an event. These events
+aren't just scattered around; they're sent to a waiting area, which we'll call
+the "Message Queue." It's like a ticket counter where events line up, waiting to
+be noticed.
+
+Now, picture specialized cooks, let's call them "watchers," keeping a keen eye
+on this waiting area. They're not just any cooks; they're experts in their
+craft. Only events that match their cooking expertise (or predefined criteria)
+make it to their attention. They don't waste time on things that aren't their
+specialty.
+
+But here's the cool part: each watcher has a unique recipe they're passionate
+about. They aren't interested in everything happening in the kitchen, just the
+events that align with their culinary skills. These events reach the watchers,
+and guess what they do? They swing into action, creating the magic in the
+kitchen – cooking up the perfect dish!
```

Comment:
This is a little redundant. I would combine these two paragraphs. Out of curiosity, was this section AI generated? I'm getting the vibes. It feels more excited than what I normally see from you.